actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -608,9 +608,9 @@ public static void getContentData(Map<String, Object> section) {
     JSONObject data;
     JSONObject jObject;
     try {
-      String baseSearchUrl = System.getenv(JsonKey.EKSTEP_BASE_URL);
+      String baseSearchUrl = System.getenv(JsonKey.SEARCH_SERVICE_API_BASE_URL);
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.